config/dms.php

```diff
@@ -1,4 +1,6 @@
 <?php
+// https://github.com/dms-org/web.laravel/issues/2
+class_alias(Dms\Web\Laravel\Action\InputTransformer\SymfonyToDmsUploadedFileTransformer::class, Dms\Web\Laravel\Action\InputTransformer\SymphonyToDmsUploadedFileTransformer::class, Dms\Web\Laravel\Action\InputTransformer\SymfonyToDmsUploadedFileTransformer::class, Dms\Web\Laravel\Action\InputTransformer\SymphonyToDmsUploadedFileTransformer::class, true);
```

Comment:
I think this was not what you were looking at.

I have a very limited experience with laravel as a framework. Browsed a bit on the package documentation and seems this config is published to app.

So in that case we may need to keep a class that extends `SymfonyToDmsUploadedFileTransformer` may work. Oh and looks like I made a mistake copying and pasting the class name to keep alias :P .